src/components/Radio/Radio.module.css

```diff
@@ -9,3 +9,88 @@
   display: flex;
   gap: var(--eds-size-1);
 }
+
+/**
+ * Wraps the visually hidden radio input element and the visible sibling svg element.
+ */
+.input__wrapper {
+  position: relative;
+  /* Centers the radio icon in the wrapper. */
+  display: inline-flex;
+  align-items: center;
+  /* Aligns the radio with the first line of the label. */
+  align-self: flex-start;
+}
+/**
+ * The visually hidden input element for the radio. The visual radio is provided by an svg element.
+ */
+.radio__input {
+  /* Removes default margins placed by browser for radioes. */
+  margin: 0;
+  /* Remove the radio from the page flow, positioning it on top of the SVG. */
+  position: absolute;
+  left: 0.25rem;
```

Comment:
nit: this should be `--eds-size-half` i think but I'm guessing this was just brought over from the other file